tests/bricks/test_conv.py

```diff
@@ -259,14 +229,14 @@ def test_convolutional_sequence():
     batch_size = 5
     activation = Rectifier().apply
 
-    conv = ConvolutionalLayer(activation, (3, 3), 5,
-                              (pooling_size, pooling_size),
-                              weights_init=Constant(1.),
-                              biases_init=Constant(5.))
+    conv = ConvolutionalActivation(activation, (3, 3), 5,
```

Comment:
Incidentally, it turns out it's completely trivial to do what I said, to allow actual elementwise `Activations` inside a `ConvolutionalSequence`. So this could be a way of reducing entropy further and eliminating `ConvolutionalActivation` as well... See https://github.com/dwf/blocks/commit/3522f51c2491d5c56cc7345443e87aa24ae959b6

But for now, let's focus on this PR.